reference/components/document-manager/creating_documents.rst

```diff
@@ -0,0 +1,93 @@
+Creating Documents
+==================
+
+The Sulu Document Manager uses interfaces to apply mapping or behaviors to
+documents (i.e. objects). These interfaces are known as *behaviors*. Behaviors
+acts upon documents.
+
+.. note::
+
+    It is equally possible to implement what is now the conventional mapping
+    pattern using XML, YAML, annotation, etc. But for now only behavioral
+    interfaces are supported.
+
+The File
+--------
+
+Because the Document Manager uses interfaces and does not depend on metadata
+mapping you can put your document anywhere you want without changing any
+configuration:
```

Comment:
`The following is an example of a document.` The other text should be in a `.. note::`.